packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -55,16 +55,37 @@ class InstrumentationRabbitMessage(RabbitMessageBase, NodeMessageBase):
     result: Optional[RunningState] = None
 
 
-class AutoscalingStatus(str, Enum):
-    IDLE = "IDLE"
-    SCALING_UP = "SCALING_UP"
-
-
-class RabbitAutoscalingMessage(RabbitMessageBase):
-    channel_name: Literal["io.simcore.autoscaling"] = "io.simcore.autoscaling"
-    origin: str
-    number_monitored_nodes: int
-    cluster_total_resources: dict[str, Any]
-    cluster_used_resources: dict[str, Any]
-    number_pending_tasks_without_resources: int
-    status: AutoscalingStatus
+class _RabbitAutoscalingBaseMessage(RabbitMessageBase):
+    channel_name: Literal["io.simcore.autoscaling"] = Field(
+        default="io.simcore.autoscaling", const=True
+    )
+    origin: str = Field(
+        ..., description="autoscaling app type, in case there would be more than one"
+    )
+
+
+class RabbitAutoscalingStatusMessage(_RabbitAutoscalingBaseMessage):
```

Comment:
Suggestion: maybe it will be interesting to have a UTC timestamp of when this was created, could help the users with understanding what is going on and help us with debugging.

Reply:
yes. I think rabbitMQ has already this integrated but I will check it out.